src/app/onboarding/components/glam/archive-type-select/archive-type-select.component.scss

```
@@ -9,7 +10,7 @@
 
   border-radius: 12px;
   border: 1px solid rgba(255, 255, 255, 0.16);
```

```suggestion
  border: none;
```
I think one of the things requested in the ticket is to remove a weird orange line that appears near the component. It seems like this change gets rid of that.